Author: Yves-Alexis Perez <corsac@debian.org>
Description: quit plymouth instead of deactivating it
plymouth is quitted from the /etc/init.d/plymouth script, but it'll hang if
plymouth has been deactivated before, so just quit it for now.
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=632737
Bug: https://bugs.launchpad.net/ubuntu/+source/lightdm/+bug/799069
diff --git a/src/display-manager.c b/src/display-manager.c
index c8f5f32..5191fcc 100644
--- a/src/display-manager.c
+++ b/src/display-manager.c
@@ -616,7 +616,7 @@ display_manager_start (DisplayManager *manager)
         plymouth_on_active_vt = plymouth_command_returns_true ("--has-active-vt");
 
         /* Deactivate stops Plymouth from drawing, as we are about to start an X server to replace it */
-        plymouth_run_command ("deactivate", NULL);
+        plymouth_run_command ("quit", NULL);
     }
 
     /* Start each static display */
